08LessonEight/homework/exercise-06.html

```diff
@@ -3,8 +3,35 @@
 
 <head>
     <title>Exercise 06</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css" media="screen" title="no title" charset="utf-8">
 </head>
 
-<body></body>
+<body>
+<div class="pictures">
+  <img src="img/truck.jpg" alt="" />
+  <img src="img/plane.jpg" alt="" />
+  <img src="img/train.jpg" alt="" />
 
-</html>
\ No newline at end of file
+</div>
+
+<div class="descriptions">
+  <div class="truck textbox">

```

Comment:
Great work! Just as a side note, I would have grouped each image with its description, would have created 3 columns a.k.a `<section>`. This would have made aligning the images and their descriptions very easy.